### PR TITLE
flatten ratings.body to a non-localised text field

### PR DESCRIPTION
**Pull request by eviljeff**
Fixes mozilla/addons#5258 ... by just dropping l10n (the solution to all our l10n problems! 😸)

### CODE REVIEW

File: src/olympia/migrations/1018-drop-l10n-from-ratings-body.sql [hunk 1]
```diff
@@ -0,0 +1,14 @@
+/* Add the new column.*/
+ALTER TABLE `reviews`
+    ADD COLUMN `text_body` longtext NULL;
```
Comment (EnTeQuAk) [inline src/olympia/migrations/1018-drop-l10n-from-ratings-body.sql]: Why not simply name this column `body`?

File: src/olympia/migrations/1018-drop-l10n-from-ratings-body.sql [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+/* Copy across the translated values.  There are some ratings that have multiple
+localizations so we're going to use latest (most recent) localization. */
+UPDATE `reviews`
+    SET `text_body` = (
+        SELECT `localized_string` FROM `translations`
+        WHERE `translations`.`id` = `reviews`.`body`
+        ORDER BY `translations`.`modified` DESC
+        LIMIT 1);
+
+/* Clean-up where old column is dropped, etc, in a subsequent migration. */
```
Comment (EnTeQuAk) [inline src/olympia/migrations/1018-drop-l10n-from-ratings-body.sql]: And separate deploy, especially the "take the most recent localization" needs some testing in prod to see that we chose the correct ones.

File: src/olympia/migrations/1018-drop-l10n-from-ratings-body.sql (hunk 1 above)
Comment (eviljeff) [reply src/olympia/migrations/1018-drop-l10n-from-ratings-body.sql]: The existing column (the fk to `translations`) is already called `body` and we can't rename that without everything blowing up during the deploy.  We could name the field in django something different - that would be a lot of renaming though to something less snappy (I took the lazy choice).

Comment (EnTeQuAk) [reply src/olympia/migrations/1018-drop-l10n-from-ratings-body.sql]: Ah, mind adding this as a comment? I'm perfectly fine with that lazy choice then :laughing:

**Pull request merged**